x-pack/solutions/security/plugins/security_solution/public/detections/components/alert_summary/table/table_section.tsx

```diff
@@ -57,29 +77,38 @@ export const TableSection = memo(({ dataView }: TableSectionProps) => {
   );
 
   const renderChildComponent = useCallback(
-    (groupingFilters: Filter[]) => <Table dataView={dataView} groupingFilters={groupingFilters} />,
-    [dataView]
+    (groupingFilters: Filter[]) => (
+      <Table
+        dataView={dataView}
+        groupingFilters={groupingFilters}
+        packages={packages}
+        ruleResponse={ruleResponse}
+      />
+    ),
+    [dataView, packages, ruleResponse]
   );
 
   return (
-    <div data-test-subj={GROUPED_TABLE_TEST_ID}>
-      <GroupedAlertsTable
-        accordionButtonContent={groupTitleRenderers}
-        accordionExtraActionGroupStats={accordionExtraActionGroupStats}
-        defaultGroupingOptions={groupingOptions}
-        from={from}
-        globalFilters={filters}
-        globalQuery={globalQuery}
-        hasIndexMaintenance={hasIndexMaintenance ?? false}
-        hasIndexWrite={hasIndexWrite ?? false}
-        loading={false}
-        renderChildComponent={renderChildComponent}
-        runtimeMappings={runtimeMappings}
-        signalIndexName={indexNames}
-        tableId={TableId.alertsOnAlertSummaryPage}
-        to={to}
-      />
-    </div>
+    <TableSectionContextProvider packages={packages} ruleResponse={ruleResponse}>
+      <div data-test-subj={GROUPED_TABLE_TEST_ID}>
+        <GroupedAlertsTable
+          accordionButtonContent={groupTitleRenderers}
+          accordionExtraActionGroupStats={accordionExtraActionGroupStats}
+          defaultGroupingOptions={groupingOptions}
+          from={from}
+          globalFilters={filters}
+          globalQuery={globalQuery}
+          hasIndexMaintenance={hasIndexMaintenance ?? false}
+          hasIndexWrite={hasIndexWrite ?? false}
+          loading={false}
```

Comment:
Is loading handled at a higher level?

Reply:
Yup

Reply:
But this could potentially be revisited in the future...